main.typ

```diff
@@ -40,6 +40,20 @@
 
 = Typical Open Source Development
 
+Open source software development is a collaborative effort between developers and users. The users of the software are able to contribute to the development of the software by reporting bugs, suggesting features, and even contributing code. The developers of the software are able to use the feedback from the users to improve the software. Open source software development differs from traditional software development in two major ways. The first being that the users are able to contribute to the development of the software. The second being that almost all collaboration between developers happens asynchronously, through the internet.
+
+Often times, contributors to open source software development are completely anonymous. This is because the contributors are not required to provide any personal information when contributing to the software. This is a major difference from traditional software development, where the developers are usually working in the same location and are able to communicate face to face. It creates a completely different dynamic when managing a project.
+
+== Project Management in Open Source Software
+
+For many smaller open source projects, the project is managed by a "Benevolent Dictator For Life", where the goals and priorities of the project are set by a single person. This same person is also responsible for making the final decisions on what features are added to the project. They take ownership of the code, and are completely responsible for it. This is a very simple way to manage a project, but it does not scale well. As the project grows, it becomes more difficult for a single person to manage the project. In mid-sized open source projects, the project may be managed by a small group of people, but under the same principles as the "Benevolent Dictator For Life".
```

Comment:
- Many smaller open source projects are managed by a "Benevolent Dictator for Life." Goals and priorities of a project are determined by one individual.
- This person is also responsible for.. 
- Ownership and responsibility of the code and project is acquired by the "Benevolent Dictator For Life."
- As the scale of the project increases... the difficulty of managing a project by one person also increases.